samples/advanced-csharp/Canary.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace AdvancedSampleWebApp
+{
+    public class Canary
+    {
+        public const int KeyLength = 32;
+        public const int NonceLength = 32;
+        public static TimeSpan ExpirationSpan = TimeSpan.FromHours(24);
```

Comment:
Readonly

Reply:
Arguably this could be a runtime parameter.